README.md

```diff
@@ -118,10 +114,12 @@ If you used TDMS in your research and found it helpful, please cite this work.
 
 ```bibtex
 @software{tdms,
-    author  = {Munro, Peter and others},
-    license = {GPL-3.0},
-    title   = {{TDMS - The Time-Domain Maxwell Solver}},
-    URL     = {https://github.com/UCL/TDMS}
+    author       = {Munro, Peter and others},
+    license      = {GPL-3.0},
+    title        = {{TDMS - The Time-Domain Maxwell Solver}},
+    URL          = {https://github.com/UCL/TDMS},
+    publisher    = {Zenodo},
+    doi          = {10.5281/zenodo.7950604}
```

Comment:
Just for info. The DOI is linked to a version. So this will become incorrect when we tag v1.1 (?) or v2 or whatever.
Arguably we should just trust the badge which auto-updates.

Reply:
Use [10.5281/zenodo.7950603](https://doi.org/10.5281/zenodo.7950603)

Reply:
You're my hero. 🧙

And also, you read the (quite clear) documentation.